app/controllers/hotspots_controller.rb

```diff
@@ -11,29 +15,28 @@ def new
     
     def create
         @hotspot = Hotspot.create(hotspot_params)
-
         if @hotspot.valid?
             flash[:notice] = "Hotspot created succesfully."
         else
             flash[:notice] = "Hotspot could not be saved."
         end
-        
+        @hotspot.save
         redirect_to hotspots_path
     end
     
     def edit
         @hotspot = Hotspot.find(params[:id])
+        @hotspot.save
```

Comment:
Why are we saving this here? We haven't made any changes. We just found a hotspot.